testcases/PowerNVDump.py

```diff
@@ -619,9 +619,11 @@ def setup_fadump(self):
         '''
         self.cv_SYSTEM.set_state(OpSystemState.OS)
         if self.distro == "rhel":
+            self.c.run_command("dnf install -y ServiceReport; servicereport -r -p kdump;", timeout=240)
```

Comment:
@SACHIN-BAPPALIGE is redhat  shipping ServiceReport rpm ?

Reply:
checking with development team @sourabhjains

Reply:
@SACHIN-BAPPALIGE ServiceReport expect user to reboot the machine if crashkernel reservation changes.

Are we taking case of such provision in the test?

Reply:
ServiceReport is not available on RHEL yet. Getting it from GitHub is the way.

Reply:
> ServiceReport is not available on RHEL yet. Getting it from GitHub is the way.

@sourabhjains : Thanks for the suggestion . Then I will go with this logic 

```
git clone https://github.com/linux-ras/ServiceReport
cd ServiceReport
python ./servicereport --plugins kdump package --repair

```